docs/getting-started/getting-started-http-06-headers.md

```diff
@@ -0,0 +1,70 @@
+---
+id: getting-started-http-06-headers
+title: Headers
+---
+
+# Headers
+
+Headers are used to pass additional information with the request or response. Model properties and parameters that should be passed in a header use the `@header` decorator. The decorator takes the header name as a parameter. If a header name is not provided, it is inferred from the property or parameter name.
+
+## What is `etag`?
```

Comment:
Is it necessary to go over what an eTag header here is? If the goal is to just add example on how to specify eTag mnaybe could have an appendix at the end to explain what it is instead of starting with it.

Reply:
I think etag has a place in the expanded context, let me know what you think.